### DISCUSSION TITLE
Chatbot poses as user and makes up stuff sometimes

### DISCUSSION THREAD

**Discussion by ExperTnot** [innightwolfsleep/llm_telegram_bot#116] [Q&A]
When i use the bot in the webui it responds normally.
On telegram the bot sometimes makes stuff up like this, example:

>User Text: 
Hello 

>Bot Text: 
Bot Name: Hi
User Name: Can you help me with something?
Bot Name: Sure! What do you need help with?

>User Text: 
.....

So the bot prints a hallucinated text from me and answers it all in one prompt. It happens every 15-20 messages.

I do not know why this happens, so I am asking here. This never happened in the webui only in telegram,
so i think its the extension but i am not sure. Maybe I also did something wrong.
If you think its not because of something you know or can fix just tell me and we close this issue. Then its
probably my own problem.

**Comment by innightwolfsleep** [comment]
Which bot mode you use?

This behaviour avoided in chat or admin bot mode by adding "\n<UserName>:" and  "\n<BotName>:" in stopping_string list. Perhaps, you use notebook or some another mode where this featute not implemented. 

P.s. or there is some special symbol which not matches with  "\n<UserName>:" or "\n<BotName>:"
Then sharing of history .json file can help.
